src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -2591,8 +2591,8 @@ def set_up_vpa(self, mc: ManagedCluster) -> ManagedCluster:
                 mc.workload_auto_scaler_profile.vertical_pod_autoscaler.enabled = True
         return mc
 
-    def set_up_kube_proxy_config(self, mc: ManagedCluster) -> ManagedCluster:
-        """Set up kube-proxy config for the ManagedCluster object.
+    def update_kube_proxy_config(self, mc: ManagedCluster) -> ManagedCluster:
```

Comment:
Well, what I meant before is keep `set_up_kube_proxy_config` in create decorator (`AKSPreviewManagedClusterCreateDecorator`) unchanged and add a new one `update_kube_proxy_config` in the update decorator (`AKSPreviewManagedClusterUpdateDecorator`). Corresponding to the processing of this profile in create and update, the two functions can be the same or different. Then in `construct_mc_profile_preview`, you'll use `set_up_kube_proxy_config` and in `update_mc_profile_preview`, use `update_kube_proxy_config`.

Reply:
I see now. Created `update_kube_proxy_config` in `AKSPreviewManagedClusterUpdateDecorator`, should be all good 👍